tools/MINGW-packages/mingw-w64-natron-setup/Natron.iss

```diff
@@ -57,7 +57,9 @@ Source: "{src}\bin\*"; DestDir: "{app}\bin\"; Components: natron; Flags: ignorev
 Source: "{src}\docs\*"; DestDir: "{app}\docs\"; Components: natron; Flags: ignoreversion recursesubdirs createallsubdirs external
 Source: "{src}\lib\*"; DestDir: "{app}\lib\"; Components: natron; Flags: ignoreversion recursesubdirs createallsubdirs external
 
-Source: "{src}\Plugins\PySide\*"; DestDir: "{app}\Plugins\PySide\"; Components: natron; Flags: ignoreversion recursesubdirs createallsubdirs external
+Source: "{src}\Plugins\PySide\*"; DestDir: "{app}\Plugins\PySide\"; Components: natron; Flags: ignoreversion recursesubdirs createallsubdirs external skipifsourcedoesntexist
```

Comment:
Added skipifsourcedoesntexist so that a single file can be used for both Qt4 and Qt5 builds.